api/catalog/settings.py

```diff
@@ -145,7 +145,7 @@
     "DEFAULT_VERSIONING_CLASS": "rest_framework.versioning.URLPathVersioning",
     "DEFAULT_RENDERER_CLASSES": (
         "rest_framework.renderers.JSONRenderer",
-        "rest_framework.renderers.BrowsableAPIRenderer",
```

Comment:
I'm curious to know why not turn off the browsable API in production.

Reply:
We discussed in Slack, but generally we find it quite useful while browsing the API in a browser!

Reply:
In production?